app/frontend/tests/unit/store/modules/user.actions.spec.js

```diff
@@ -0,0 +1,39 @@
+import { cloneDeep } from 'lodash';
+import { createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
+
+import userService from '@/services/userService';
+import userStore from '@/store/modules/user';
+
+describe('user.js - getUserAcronyms action', () => {
+  let store;
+  beforeEach(() => {
+    const localVue = createLocalVue();
+    localVue.use(Vuex);
+
+    store = new Vuex.Store(cloneDeep(userStore));
+  });
+
+  const spy = jest.spyOn(userService, 'getUserAcronyms');
+  jest.mock('../../../../src/services/userService');
```

Comment:
This line might not be needed as you're already spying on getUserAcronyms (which by definition forces that function to be mock intercepted)

Reply:
Ah didn't know, thanks